python/social_bot/envs/grocery_ground.py

```diff
@@ -480,11 +492,13 @@ def __init__(self,
 
         self._teacher = teacher.Teacher(task_groups_exclusive=False)
         if task_name is None or task_name == 'goal':
+            if random_goal is None:
+                random_goal = with_language
             main_task = GroceryGroundGoalTask(
-                max_steps=200,
+                max_steps=max_steps,
```

Comment:
We can remove 498-502, and make the default parameter of GroceryGroundGoalTask the same as the initialization here. 
We can config GroceryGroundGoalTask through gin when needed.

Reply:
Good point!  Done.